gb/Cargo.toml

```diff
@@ -20,9 +21,12 @@ crossbeam-channel = "0.5"
 cfg-if = "1.0"
 crossterm = "0.23"
 rppal = {version = "0.13", optional = true}
+libc = "0.2"
+nix = {version = "0.24", features = ["ioctl"]}
+fast_image_resize = "0.9.3"
```

Comment:
Remove the fir library, on arm my c and rust bi-linear interpolations are good enough